engine/src/main/java/org/hibernate/search/analyzer/impl/LuceneEmbeddedAnalyzerStrategy.java

```diff
@@ -35,7 +35,7 @@
  */
 public class LuceneEmbeddedAnalyzerStrategy implements AnalyzerStrategy {
 
-	private static final Log log = LoggerFactory.make();
+	private static final Log LOG = LoggerFactory.make();
```

Comment:
True, we could consider that loggers are output streams, and as such they can be considered "mutable" as `System.out` for instance.
I've always followed the rule "static final => capital case", but there's no real reason beyond consistency. And it seems I was wrong in thinking "LOG" is more common: we have 163 occurrences of "static final Log log" vs. 72 occurrences of "static final Log LOG".
Maybe I should try and rename all those loggers to "log"... one day... In the meantime, I'll remember to name them "log".